fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -571,17 +611,14 @@ public List<MaterializedView> getMaterializedViews() {
     public Set<String> getTableNamesWithLock() {
         readLock();
```

Comment:
i think we don't need db read lock here any more because we just return a ref to TableStore.nameSet

Reply:
I think there is still a need for a lock here.